src/Symfony/Component/PropertyInfo/Extractor/PhpStanExtractor.php

```diff
@@ -305,4 +312,9 @@ private function getPhpDocNode(string $rawDocNode): PhpDocNode
 
         return $phpDocNode;
     }
+
+    public function canAccessMemberBasedOnItsVisibility(\ReflectionProperty|\ReflectionMethod $member): bool
+    {
+        return $member->isPublic() || $this->allowPrivateAccess;
```

Comment:
```suggestion
        return $this->allowPrivateAccess || $member->isPublic();
```


Same, prevents a useless function call if `allowPrivateAccess` is true.